monitor/cron_monitor.sh

```diff
@@ -47,7 +47,7 @@ echo "Hours since mediawiki Dump: $hours_since_mediawiki_dump"
 echo "Hours since sql Dump: $hours_since_sql_dump"
 
 # check if the backup is within the required timeframe 
-minimum_backup_time_requirement=1.5 
+minimum_backup_time_requirement=$EXPECTED_BACKUP_TIMELAG
```

Comment:
Can we set the default value if the env is not exists?

Something like `${EXPECTED_BACKUP_TIMELAG:-1.5}` this.